src/main/java/com/softserveinc/ita/mmakoviichuk/pageobjects/softserve/SoftServePage.java

```diff
@@ -1,17 +1,18 @@
 package com.softserveinc.ita.mmakoviichuk.pageobjects.softserve;
 
+import com.codeborne.selenide.SelenideElement;
 import org.openqa.selenium.WebElement;
 
+import static com.codeborne.selenide.Selenide.$x;
 import static com.softserveinc.ita.mmakoviichuk.utils.runners.ElementsUtil.*;
 import static java.lang.String.format;
-import static org.openqa.selenium.By.xpath;
 
 public class SoftServePage {
 
-    private WebElement sidebarSection;
+    private SelenideElement sidebarSection;
 
     public void switchSidebarSection (String section) {
-        sidebarSection = $x(xpath(format("//a[div[@class = 'side-navigation__title' and text() = '%s']]", section)));
+        sidebarSection = $x(format("//a[div[@class = 'side-navigation__title' and text() = '%s']]", section));
```

Comment:
$x(format("//a[div[@class = 'side-navigation__title' and text() = '%s']]", section)).click();